File: Python/tests/test_hgf_ibrb_single.py
import pytest

from hbayesdm.models import hgf_ibrb_single


def test_hgf_ibrb_single():
    _ = hgf_ibrb_single(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_hgf_ibrb_single.R
context("Test hgf_ibrb_single")
library(hBayesDM)

test_that("Test hgf_ibrb_single", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(hgf_ibrb_single(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
